dev/com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateDataSourceTest.java

```diff
@@ -329,22 +313,6 @@ public void testMultipleWithNoResults() throws Exception {
         assertEquals("unexpected response: " + json, 0, json.size());
     }
 
-    @Test
-    public void testNestedUnderTransaction() throws Exception {
-        JsonObject json = new HttpsRequest(server, "/ibm/api/validation/dataSource/transaction%2FdataSource[default-0]?auth=container").run(JsonObject.class);
```

Comment:
Same as previous comments - do not remove the only tests of dataSource nested under singleton pid.